src/main/resources/META-INF/jpa-schema-export.sql

```diff
@@ -1078,8 +1078,8 @@
        references Word (id);
 
     alter table Word_LetterSoundCorrespondence 
-       add constraint FKf6r3yfdc6quwa0b13mln5uuc8 
-       foreign key (letterSoundCorrespondences_id) 
```

Comment:
There is a mismatch between the `jpa-schema-export.sql` and the production database:

```sql
MariaDB [webapp-HIN]> SHOW CREATE TABLE Word_LetterSoundCorrespondence;
```
```sql
| Word_LetterSoundCorrespondence | CREATE TABLE `Word_LetterSoundCorrespondence` (
  `Word_id` bigint(20) NOT NULL,
  `letterSoundCorrespondences_id` bigint(20) NOT NULL,
  `letterSoundCorrespondences_ORDER` int(11) NOT NULL,
  PRIMARY KEY (`Word_id`,`letterSoundCorrespondences_ORDER`),
  KEY `FK_aegghkhls25m6pr0e42f1csvi` (`letterSoundCorrespondences_id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
```

Reply:
The foreign key remained the same after the DB migration:

```sql
MariaDB [webapp-HIN]> SHOW CREATE TABLE Word_LetterSoundCorrespondence;
```
```sql
| Word_LetterSoundCorrespondence | CREATE TABLE `Word_LetterSoundCorrespondence` (
  `Word_id` bigint(20) NOT NULL,
  `letterSounds_id` bigint(20) NOT NULL,
  `letterSounds_ORDER` int(11) NOT NULL,
  PRIMARY KEY (`Word_id`,`letterSounds_ORDER`),
  KEY `FK_aegghkhls25m6pr0e42f1csvi` (`letterSounds_id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
```

Reply:
Removed the `DROP FOREIGN KEY` from the migration script: https://github.com/elimu-ai/webapp/pull/1716/commits/aafe7dd1143e8a8ecc6fecf668d0e7ab47d9bf2d